test-e2e/requirements.txt

```diff
@@ -1,9 +1,9 @@
-git+https://github.com/dcos/dcos-e2e.git@2020.05.26.0
+git+https://github.com/dcos/dcos-e2e.git@2020.12.18.0
 attrs==19.1.0
 cryptography==3.2.1
 docker==4.0.2
 etcd3==0.11.1
-jwt==0.5.4
+PyJWT==1.7.1
```

Comment:
So historically it looks like when pip was using the legacy resolver, PyJWT was getting installed from a downstream dep somewhere *after* it had installed jwt here, so the effective pacakge of `import jwt` was PyJWT. Now that pip is using the new resolver, `import jwt` is getting jwt, which has a different api. :facepalm:

Reply:
That's an interesting find!